packages/docusaurus-plugin-content-blog/src/index.ts

```diff
@@ -267,25 +285,22 @@ export default function pluginContentBlog(
         }),
       );
 
-      blogItems.forEach((blogItem, index) => {
-        const prevItem = index > 0 ? blogItems[index - 1] : null;
-        const nextItem =
-          index < blogItems.length - 1 ? blogItems[index + 1] : null;
-        const {metadata, metadataPath} = blogItem;
-        const {source, permalink} = metadata;
+      await Promise.all(
```

Comment:
async map is faster wohooo

Reply:
There's nothing to `await` inside, why `async`? async without await inside is as good as synchronous. I'm surprised there are no lint rules for this. Please change it back.

Reply:
Oops. you're correct. I had this async before because of older change I made, after refactor i forget to change it back.